docs/DeveloperGuide.md

```diff
@@ -895,3 +895,32 @@ testers are expected to do more *exploratory* testing.
 
 1. Test case: `exit`<br>
        Expected: The app closes.
+
+## **Appendix: Effort**
+
+### Difficulty Level
+VendorVault is significantly more complex than Address Book 3 (AB3). While AB3 deals with a single entity (Person), 
+VendorVault deals with two entities (Supplier and Delivery), thus more effort is required to manage and integrate the two entities seamlessly. 
+Apart from introducing new commands, the UI was also improved to support displaying both entities in a user-friendly manner.
+
+### Challenges Faced
```

Comment:
Can we write about having a separate list for sorting as a challenge?

Reply:
I think it is ok